### PR TITLE
fix: Activity panel and notification screen design tweaks

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

This PR does some design updates the new Activity screen and the notification pages:

* Updating the Notification artwork list component, mainly to increase the image size and have consistency with the New Works for You screen that displays a similar artwork list.
* Adding the "Following" Activity panel filter pill (with empty message)
* Adding the "Edit Follows" button
* Design tweaks (spacing etc.)

| | | | 
| --- | --- | --- |
|Simulator Screenshot - iPhone 15 Pro - 2024-02-21 at 12 14 27 | Simulator Screenshot - iPhone 15 Pro - 2024-02-21 at 12 14 11 | Simulator Screenshot - iPhone 15 Pro - 2024-02-21 at 12 14 23|

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Update Activity panel & notification page designs (behind feature flag) - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1d0e57533cb15eeeba7cb984e592c7dbfcf3b0db
</p>

**Review by MounirDhahri — APPROVED** [review]
LGTM

File: src/app/Scenes/Activity/components/ArtworkPublishedNotification.tsx [hunk 1]
```diff
@@ -52,6 +53,12 @@ export const ArtworkPublishedNotification: FC<ArtworkPublishedNotificationProps>
     })
   }
 
+  const handleEditFollowsButton = () => {
+    // TODO: Add tracking
+
+    navigate("/favorites")
+  }
```
Comment (dariakoko) [inline src/app/Scenes/Activity/components/ArtworkPublishedNotification.tsx:60]: Hey @olerichter00 
Have you and Barney decided to add the Follows button back? 
See the comments from Barney last week here: https://www.notion.so/artsy/I-don-t-think-we-need-the-Edit-Follows-CTA-at-the-bottom-of-the-Follows-page-e8477f40a87a497a816b8f050d06dd33?pvs=4

File: src/app/Scenes/Activity/components/NotificationArtworkList.tsx [hunk 2]
```diff
@@ -15,56 +14,42 @@ interface NotificationArtworkListProps {
...
-              showSaveIcon
-              size="fullWidth"
-              metaContainerStyles={{
-                paddingHorizontal: space(2),
-              }}
-              priceOfferMessage={priceOfferMessage}
-            />
-          )
-        }}
+      <MasonryInfiniteScrollArtworkGrid
```
Comment (dariakoko) [inline src/app/Scenes/Activity/components/NotificationArtworkList.tsx:25]: 👏 👏 👏  thanks for addressing it

**Pull request merged**